store/playbackManager.ts

```diff
@@ -269,6 +276,17 @@ export const actions: ActionTree<PlaybackManagerState, PlaybackManagerState> = {
   changeCurrentTime({ commit }, { time }: { time: number | null }) {
     commit('CHANGE_CURRENT_TIME', { time });
   },
+  skipForward({ commit, state }) {
+    commit('CHANGE_CURRENT_TIME', { time: (state.currentTime || 0) + 15 });
+  },
+  skipBackward({ commit, state }) {
+    // TODO: Store time to skip in a store to make it customizable
```

Comment:
I'd already had it for now in the state and its default values

Reply:
I meant it'll be stored in the user playback preferences store which hasn't been implemented yet

Reply:
Fair enough :)